Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -303,14 +303,10 @@ - (BOOL)updateAvatarView
         [OWSPrimaryStorage.sharedManager.dbReadConnection readWithBlock:^(YapDatabaseReadTransaction *transaction) {
             groupChat = [LKDatabaseUtilities getGroupChatForThreadID:self.viewItem.interaction.uniqueThreadId transaction: transaction];
         }];
-        if (groupChat != nil) {
-            BOOL isModerator = [LKGroupChatAPI isUserModerator:incomingMessage.authorId forGroup:groupChat.channel onServer:groupChat.server];
-            UIImage *moderatorIcon = [UIImage imageNamed:@"Crown"];
-            self.moderatorIconImageView.image = moderatorIcon;
-            self.moderatorIconImageView.hidden = !isModerator;
-        } else {
-            self.moderatorIconImageView.hidden = YES;
-        }
+        BOOL isModerator = [LKGroupChatAPI isUserModerator:incomingMessage.authorId forGroup:groupChat.channel onServer:groupChat.server];
```

Comment:
`groupChat` could be `nil` here

Reply:
Not actually, right? Unless we made a mistake somewhere?

Reply:
`getGroupChatForThreadID:` returns nil if a group chat object doesn't exist. This may be the case for private group chats.

Reply:
Okay, yeah, it makes sense if you're thinking about private group chats already

Reply:
We should get on the same page about public chats / private group chats / private chats

Reply:
`LokiGroupChat` should really be `LokiPublicChat` then

Reply:
I'll refactor this now so we don't get confused anymore